pages/am/3.x/user-guide/user-management/user/user-guide-users.adoc

```diff
@@ -8,11 +8,117 @@
 
 === List users
 
-To search users, navigate to `Users` section of your domain's settings. The input search box will let you find user by its username.
+To search users, navigate to `Users` section of your domain's settings and set a search query expression.
+You can also list users via the `/users` endpoint of the AM management API.
 
 image::am/current/graviteeio-am-userguide-um-search-users.png[]
 
-NOTE: Currently only users created by the AM management API or the SCIM protocol and external users who have been already logged can be found.
+NOTE: Currently only users created by the AM management API or the SCIM protocol and external users who have been already logged in can be found.
```

Comment:
NOTE: You can only list users created with AM API or the SCIM protocol and external users who have already logged in.